### PR TITLE
Update Topology view to use @patternfly/react-topology components

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
This changes the topology view in developer mode to use the common topology view components in the @patternfly/react-topology package.

https://jira.coreos.com/browse/ODC-1002
https://jira.coreos.com/browse/ODC-1079

### CODE REVIEW

File: frontend/packages/dev-console/src/components/topology/TopologySideBar.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { ModelessOverlay } from 'patternfly-react';
+import { TopologySideBar as PfTopologySideBar } from '@patternfly/react-topology';
```
Comment (spadgett) [inline frontend/packages/dev-console/src/components/topology/TopologySideBar.tsx]: Console convention is to use all uppercase for acronyms in camel case names

```suggestion
import { TopologySideBar as PFTopologySideBar } from '@patternfly/react-topology';
```

**Comment by jeff-phillips-18** [comment]
Update the overrides for page font sizes to include the pf-l-stack layout (was causing the larger font in the dialog).

**Comment by christianvogt** [comment]
@jeff-phillips-18 I very much dislike how layout components of PF affect font sizes.

File: frontend/public/style/_overrides.scss [hunk 2]
```diff
@@ -158,6 +158,9 @@ kbd {
     font-size: $font-size-base;
     height: auto;
   }
+  .pf-l-stack {
+    font-size: $font-size-base;
```
Comment (christianvogt) [inline frontend/public/style/_overrides.scss]: @spadgett didn't we talk about changing the default font size globally?
cc @sg00dwin

File: frontend/packages/dev-console/src/components/topology/Topology.tsx [hunk 3]
```diff
@@ -44,30 +44,52 @@ export default class Topology extends React.Component<TopologyProps, State> {
...
-      <GraphToolbarButton label="Reset Layout" onClick={graphApi.resetLayout}>
-        <ExpandIcon />
-      </GraphToolbarButton>
-    </GraphToolbar>
-  );
+  graphApiRef = (api: GraphApi) => {
+    this.setState({ graphApi: api });
+  };
+
+  renderControlBar = () => {
```
Comment (christianvogt) [inline frontend/packages/dev-console/src/components/topology/Topology.tsx]: I'd prefer that we now move the control bar to a separate component file.

File: frontend/packages/dev-console/src/components/topology/Graph.tsx [hunk 4]
```diff
@@ -29,6 +29,7 @@ export interface GraphProps {
   children?(GraphApi): React.ReactNode;
```
Comment (christianvogt) [inline frontend/packages/dev-console/src/components/topology/Graph.tsx]: This isn't needed anymore with your changes.

File: frontend/public/style/_overrides.scss (hunk 2 above)
Comment (spadgett) [reply frontend/public/style/_overrides.scss]: > @spadgett didn't we talk about changing the default font size globally?

Yes, @sg00dwin is still looking at it. It sounds more involved than just updating one var unfortunately.

**Comment by spadgett** [comment]
/lgtm cancel

as @christianvogt has review feedback

**Comment by spadgett** [comment]
/assign @christianvogt 
/approve
pending review comments

File: frontend/public/style/_overrides.scss (hunk 2 above)
Comment (christianvogt) [reply frontend/public/style/_overrides.scss]: ok for now we will go with this change then

**Pull request merged**